packages/miniflare/src/plugins/shared/constants.ts

```diff
@@ -58,3 +70,14 @@ export function objectEntryWorker(
 		],
 	};
 }
+
+// Value of `unsafeUniqueKey` that forces the use of "colo local" ephemeral
```

Comment:
Ephemeral being the key term in all of this right?
I.e. they are ephemeral because they can't persist state between instances?